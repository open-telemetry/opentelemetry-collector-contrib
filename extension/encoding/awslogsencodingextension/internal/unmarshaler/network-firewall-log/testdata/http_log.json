{
    "firewall_name": "test-firewall",
    "availability_zone": "us-east-1a",
    "event_timestamp": "2025-10-20T10:30:45.123Z",
    "event": {
        "event_type": "http",
        "flow_id": 1234567893,
        "src_ip": "192.168.1.100",
        "src_port": 54324,
        "dest_ip": "10.0.1.53",
        "dest_port": 80,
        "proto": "TCP",
        "http": {
            "hostname": "example.com",
            "url": "/api/v1/users",
            "http_user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36",
            "http_content_type": "application/json",
            "cookie": "session_id=abc123; user_pref=dark_mode"
        }
    }
}
