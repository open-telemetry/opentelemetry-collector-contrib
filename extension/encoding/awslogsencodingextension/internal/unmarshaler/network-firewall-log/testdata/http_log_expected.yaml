resourceLogs:
  - resource:
      attributes:
        - key: cloud.provider
          value:
            stringValue: aws
        - key: aws.networkfirewall.name
          value:
            stringValue: test-firewall
        - key: cloud.availability_zone
          value:
            stringValue: us-east-1a
    scopeLogs:
      - scope:
          name: github.com/open-telemetry/opentelemetry-collector-contrib/extension/encoding/awslogsencodingextension
          version: ""
          attributes:
            - key: encoding.format
              value:
                stringValue: aws.networkfirewall
        logRecords:
          - observedTimeUnixNano: "0"
            timeUnixNano: "1760956245123000000"
            attributes:
              - key: aws.networkfirewall.event.type
                value:
                  stringValue: http
              - key: aws.networkfirewall.flow_id
                value:
                  intValue: "1234567893"
              - key: source.address
                value:
                  stringValue: 192.168.1.100
              - key: source.port
                value:
                  intValue: "54324"
              - key: destination.address
                value:
                  stringValue: 10.0.1.53
              - key: destination.port
                value:
                  intValue: "80"
              - key: network.transport
                value:
                  stringValue: TCP
              - key: url.domain
                value:
                  stringValue: example.com
              - key: url.path
                value:
                  stringValue: /api/v1/users
              - key: user_agent.original
                value:
                  stringValue: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36
              - key: http.request.header.content-type
                value:
                  stringValue: application/json
              - key: http.request.header.cookie
                value:
                  stringValue: session_id=abc123; user_pref=dark_mode
